external/mbedtls/Makefile

```diff
@@ -86,6 +86,7 @@ SRC_TLS_CSRCS =       debug.c         net.c           ssl_cache.c     \
 
 ifeq ($(CONFIG_TLS_WITH_SSS),y)
 SRC_SEE_CSRCS += see_api.c	see_internal.c
+CFLAGS+=-I$(TOPDIR)/arch/arm/src/s5j/sss
 endif
```

Comment:
1. Is it really allowed to add soc specific in external modules. If then then how will you manage multiple soc. currently looks like you restrict to only s5j. Please think of common approach
2. This becomes more complex or compilation will break to handle when the soc changes.
3. I don't think adding chip specific headers is good into external modules. maintainance is difficult. Instead there should be one common header file in include/tinyara/x.h
This x.h should include
chip/y.h

Reply:
when I say chip/y.h it literally means chip.h. Not s5j/y.h